### DISCUSSION TITLE
Windows Servers generating random domains that causing security alerts

### DISCUSSION THREAD

**Discussion by kumarvemula2** [community/community#113706] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

We have windows servers that generating random domain names that causing security alerts as DGA. Below is the document we got...and we tested this we saw that  servers generating alerts when there is a restart in server or network service. 

https://learn.microsoft.com/en-us/troubleshoot/windows-client/networking/dns-requests-random-network-properties-change

we want to know why what reason it generating these domains. so that we can try to fix that problem.

**Comment by ettaboyle** [comment]
Hi @kumarvemula2,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by kumarvemula2** [reply]
I posted this here as I am not sure how to connect to people who contributed that document. They mentioned their names and linked to github.
 can anyone help me how to reach them? you can see them on the document under contributors.
